robust-websocket.js

```diff
@@ -93,12 +95,17 @@
     }
 
     self.open = function() {
-      if (realWs.readyState !== WebSocket.OPEN && realWs.readyState !== WebSocket.CONNECTING) {
+      if (realWs.readyState !== self.OPEN && realWs.readyState !== self.CONNECTING) {
```

Comment:
We don't and should not need to reimplement logic that is in the underlying `WebSocket.prototype.send` method. if it's missing, that's another thing and in that case, throwing an `Error` indicating it's missing is the most explicit, best experience for the user.